api/src/main/resources/openmrs-distro.properties

```diff
@@ -52,5 +52,9 @@ omod.orderentryui=${orderentryuiVersion}
 owa.openmrs-owa-cohortbuilder=${cohortBuilderOwaVersion}
 omod.mirebalais=${project.parent.version}
 db.h2.supported=false
+property.pih.config.prompt=Please specify the PIH config files to use
+property.pih.config.default=mirebalais,mirebalais-humci
+property.pih.config.dir.prompt=Please provide the path to the PIH config files
+property.pih.config.dir.default=/etc/puppet/mirebalais-modules/openmrs/files/config/
```

Comment:
We con't need this config directory anymore, these are now just in .OpenMRS/configuration.